pom.xml

```diff
@@ -31,6 +33,17 @@
         <version.slf4j>1.7.25</version.slf4j>
     </properties>
 
+    <distributionManagement>

```

Comment:
I would not add that distributionManagement pointing to the PSI nexus in a publicly available repository. Instead, use custom maven settings to do so.

Reply:
distributionManagement removed